src/components/MTableEditCell/index.js

```diff
@@ -72,61 +56,48 @@ function MTableEditCell(props) {
   };
 
   const onApprove = () => {
-    setState({ ...state, isLoading: true });
+    setState({ ...state, isLoading: true }, () => {
```

Comment:
setState from the useState does not have the callback which is in the class setState, so this does not work. Thats why it wa smoved into the useEffect which you removed.